index.js

```diff
@@ -36,10 +40,38 @@ module.exports = function (options) {
 				}
 			})
 			.then(function () {
+				var options = [];
+
+				if (device) {
+					options.push('--device');
+				}
+				if (release) {
+					options.push('--release');
+				}
+
 				// Build the platform
-				return cordova.build({platforms: ['ios']});
+				return cordova.build({platforms: ['ios'], options: options});
 			})
 			.then(function () {
+				var base = path.join(iosPath, 'build/device');
+				var cwd = process.env.PWD;
+
+				// Iterate over the output directory
+				fs.readdirSync(base).forEach(function (file) {
+					// Check if the file ends with .apk
+					if (file.indexOf('.ipa') !== -1) {
```

Comment:
The comment talks about '.apk', not '.ipa'

FWIW: This comment seems redundant for anyone able to read the code :)